rskj-core/src/main/java/co/rsk/net/SyncProcessor.java

```diff
@@ -118,7 +119,7 @@ public void processBodyResponse(MessageChannel peer, BodyResponseMessage message
 
     public void processNewBlockHash(MessageChannel sender, NewBlockHashMessage message) {
         logger.trace("Process new block hash from node {} hash {}", sender.getPeerNodeID(), HashUtil.shortHash(message.getBlockHash()));
-        byte[] hash = message.getBlockHash();
+        Keccak256 hash = message.getBlockHash();

```

Comment:
I'd rather search for the usage of HashUtil.shortHash() and try to change usages to the Keccak256 method.